intranet/apps/polls/models.py

```diff
@@ -94,6 +94,33 @@ def get_users_voted(self):
                 users = list(q.get_users_voted())
         return users
 
+    def get_num_eligible_voters(self):
+        if self.groups.exists():
+            q_objects = Q()
+            for g in self.groups.all():
+                q_objects.add(Q(name=g), Q.OR)
+            groups = DjangoGroup.objects.filter(q_objects)
+            users = get_user_model().objects.none()
+            for g in groups:
+                users = users.union(g.user_set.all())
+            return users.count()
+        else:
+            return get_user_model().objects.count() - 1
```

Comment:
What is the meaning of `-1` here? Please add a comment explaining it for future reference.

Reply:
There is the RESET_PASSWORD user here, that obviously isn't voting in the poll, but I think is included in the count of all the users. I will comment to clarify this.

Reply:
Hmm. In that case, we should instead use `.exclude(user_type="service")`. More generalized, and more obvious from the code itself what's going on.